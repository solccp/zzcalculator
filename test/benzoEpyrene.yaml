---
structure_id: 1
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 19]
- [2, 3]
- [16, 18]
- [17, 15]
- [10, 9]
- [12, 13]
- [14, 8]
- [7, 11]
---
structure_id: 2
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 19]
- [2, 3]
- [16, 18]
- [17, 15]
- [10, 9]
- [12, 11]
- [13, 14]
- [8, 7]
---
structure_id: 3
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 19]
- [2, 3]
- [16, 18]
- [17, 15]
- [10, 9]
rings:
- [12, 11, 7, 8, 14, 13]
---
structure_id: 4
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 19]
- [2, 3]
- [16, 9]
- [18, 17]
- [15, 10]
- [14, 13]
- [8, 7]
- [11, 12]
---
structure_id: 5
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 19]
- [2, 3]
- [16, 9]
- [18, 17]
- [15, 10]
- [14, 8]
- [13, 12]
- [7, 11]
---
structure_id: 6
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 19]
- [2, 3]
- [16, 9]
- [18, 17]
- [15, 10]
rings:
- [14, 8, 7, 11, 12, 13]
---
structure_id: 7
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 19]
- [2, 3]
- [14, 13]
- [12, 11]
- [8, 7]
rings:
- [16, 9, 10, 15, 17, 18]
---
structure_id: 8
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 19]
- [2, 3]
- [14, 8]
- [13, 12]
- [11, 7]
rings:
- [16, 9, 10, 15, 17, 18]
---
structure_id: 9
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 19]
- [2, 3]
rings:
- [16, 9, 10, 15, 17, 18]
- [14, 8, 7, 11, 12, 13]
---
structure_id: 10
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 2]
- [3, 10]
- [19, 15]
- [17, 18]
- [16, 9]
- [13, 14]
- [8, 7]
- [11, 12]
---
structure_id: 11
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 2]
- [3, 10]
- [19, 15]
- [17, 18]
- [16, 9]
- [13, 12]
- [11, 7]
- [8, 14]
---
structure_id: 12
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [20, 2]
- [3, 10]
- [19, 15]
- [17, 18]
- [16, 9]
rings:
- [13, 12, 11, 7, 8, 14]
---
structure_id: 13
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [17, 18]
- [16, 9]
- [13, 14]
- [12, 11]
- [7, 8]
rings:
- [20, 2, 3, 10, 15, 19]
---
structure_id: 14
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [17, 18]
- [16, 9]
- [13, 12]
- [14, 8]
- [11, 7]
rings:
- [20, 2, 3, 10, 15, 19]
---
structure_id: 15
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [17, 18]
- [16, 9]
rings:
- [20, 2, 3, 10, 15, 19]
- [13, 12, 11, 7, 8, 14]
---
structure_id: 16
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [3, 10]
- [4, 7]
- [15, 17]
- [11, 12]
- [18, 16]
- [13, 14]
- [8, 9]
---
structure_id: 17
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [3, 4]
- [17, 18]
- [16, 9]
- [15, 10]
- [14, 13]
- [12, 11]
- [7, 8]
---
structure_id: 18
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [3, 4]
- [17, 18]
- [16, 9]
- [15, 10]
- [14, 8]
- [7, 11]
- [12, 13]
---
structure_id: 19
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [3, 4]
- [17, 18]
- [16, 9]
- [15, 10]
rings:
- [14, 8, 7, 11, 12, 13]
---
structure_id: 20
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [3, 4]
- [17, 15]
- [18, 16]
- [10, 9]
- [14, 13]
- [8, 7]
- [11, 12]
---
structure_id: 21
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [3, 4]
- [17, 15]
- [18, 16]
- [10, 9]
- [14, 8]
- [7, 11]
- [13, 12]
---
structure_id: 22
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [3, 4]
- [17, 15]
- [18, 16]
- [10, 9]
rings:
- [14, 8, 7, 11, 12, 13]
---
structure_id: 23
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [3, 4]
- [7, 11]
- [8, 14]
- [13, 12]
rings:
- [17, 15, 10, 9, 16, 18]
---
structure_id: 24
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [3, 4]
- [7, 8]
- [11, 12]
- [14, 13]
rings:
- [17, 15, 10, 9, 16, 18]
---
structure_id: 25
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [3, 4]
rings:
- [17, 15, 10, 9, 16, 18]
- [7, 8, 14, 13, 12, 11]
---
structure_id: 26
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [11, 12]
- [16, 18]
- [17, 15]
- [14, 13]
rings:
- [3, 4, 7, 8, 9, 10]
---
structure_id: 27
number_of_atoms: 20
double_bonds:
- [20, 19]
- [7, 11]
- [12, 13]
- [14, 8]
- [18, 17]
- [16, 9]
- [10, 15]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 28
number_of_atoms: 20
double_bonds:
- [20, 19]
- [7, 11]
- [12, 13]
- [14, 8]
- [18, 16]
- [9, 10]
- [15, 17]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 29
number_of_atoms: 20
double_bonds:
- [20, 19]
- [7, 11]
- [12, 13]
- [14, 8]
rings:
- [1, 2, 3, 4, 5, 6]
- [18, 16, 9, 10, 15, 17]
---
structure_id: 30
number_of_atoms: 20
double_bonds:
- [20, 19]
- [7, 8]
- [11, 12]
- [13, 14]
- [9, 16]
- [10, 15]
- [18, 17]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 31
number_of_atoms: 20
double_bonds:
- [20, 19]
- [7, 8]
- [11, 12]
- [13, 14]
- [9, 10]
- [16, 18]
- [15, 17]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 32
number_of_atoms: 20
double_bonds:
- [20, 19]
- [7, 8]
- [11, 12]
- [13, 14]
rings:
- [1, 2, 3, 4, 5, 6]
- [9, 10, 15, 17, 18, 16]
---
structure_id: 33
number_of_atoms: 20
double_bonds:
- [20, 19]
- [9, 16]
- [18, 17]
- [15, 10]
rings:
- [1, 2, 3, 4, 5, 6]
- [7, 8, 14, 13, 12, 11]
---
structure_id: 34
number_of_atoms: 20
double_bonds:
- [20, 19]
- [9, 10]
- [15, 17]
- [18, 16]
rings:
- [1, 2, 3, 4, 5, 6]
- [7, 8, 14, 13, 12, 11]
---
structure_id: 35
number_of_atoms: 20
double_bonds:
- [20, 19]
rings:
- [1, 2, 3, 4, 5, 6]
- [7, 8, 14, 13, 12, 11]
- [9, 10, 15, 17, 18, 16]
