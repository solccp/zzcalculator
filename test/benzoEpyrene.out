 11 + 16 x + 7 x^2 + 1 x^3
 total: 35
